.github/workflows/benchmarking.yml

```diff
@@ -76,14 +83,20 @@ jobs:
       SAVE_PERF_REPORT: ${{ github.event.inputs.save_perf_report || ( github.ref_name == 'main' ) }}
       PLATFORM: ${{ matrix.PLATFORM }}
 
-    runs-on: [ self-hosted, us-east-2, x64 ]
+    runs-on: ${{ matrix.RUNNER }}
     container:
-      image: 369495373322.dkr.ecr.eu-central-1.amazonaws.com/build-tools:pinned
+      image: ${{ matrix.IMAGE }}
       options: --init
 
     steps:
     - uses: actions/checkout@v4
 
+    - name: Configure AWS credentials # necessary on Azure runners
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        aws-region: us-east-2
+        role-to-assume: ${{ secrets.DEV_AWS_OIDC_ROLE_ARN }}
+    - run: aws sts get-caller-identity
```

Comment:
This is only needed for debugging, I think. Also can be removed from https://github.com/neondatabase/neon/pull/8651/files#diff-737fb5e48b4dcf12a9d9e04bc36ff2f3bce43cf1ee94a626142e5c3f83fea6a8R358.

Reply:
will try removing it